src/include/migraphx/op/quant_convolution.hpp

```diff
@@ -19,9 +19,9 @@ namespace op {
 
 struct quant_convolution
 {
-    std::array<std::size_t, 2> padding  = {{0, 0}};
-    std::array<std::size_t, 2> stride   = {{1, 1}};
-    std::array<std::size_t, 2> dilation = {{1, 1}};
+    std::vector<std::size_t> padding  = {0, 0};
+    std::vector<std::size_t> stride   = {1, 1};
+    std::vector<std::size_t> dilation = {1, 1};
```

Comment:
The compte_shape() function needs to be updated to reflect the multiple dimensions (same as the convolution). Also, same as above, we need check to ensure dimensions to match among inputs and attributes.

Reply:
I didn't make any changes to quant_convolution aside from making the array to vector in case there were any side effects. I will change accordingly.